lib/framebuffer.cc

```diff
@@ -39,7 +39,10 @@ static const long kBaseTimeNanos = 200;
 // This is just the negative of SUPPORT_MULTI_PARALLEL, but conceptually
 // it is something different as it means some 'classic' GPIO pins have a
 // different function. So make it a separate #define for readability.
-#ifdef SUPPORT_MULTI_PARALLEL
+#ifdef ADAFRUIT_RGBMATRIX_HAT
+#  undef SUPPORT_MULTI_PARALLEL
+#endif
+#if   defined(SUPPORT_MULTI_PARALLEL) || defined(ADAFRUIT_RGBMATRIX_HAT)
```

Comment:
In this line, now you don't need the defined(ADAFRUIT_RGBMATRIX_HAT). So this can go back to
#ifdef SUPPORT_MULTI_PARALLEL